pkgs/by-name/id/idb-companion/package.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchurl
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "idb-companion";
+  version = "1.1.8";
+
+  src = fetchurl {
+    url = "https://github.com/facebook/idb/releases/download/v${finalAttrs.version}/idb-companion.universal.tar.gz";
+    hash = "sha256-O3LMappbGiKhiCBahAkNOilDR6hGGA79dVzxo8hI4+c=";
+  };
+
+  unpackPhase = ''
+    runHook preUnpack
+
+    tar -xzf $src
+
+    runHook postUnpack
+  '';
```

Comment:
To be confirmed, but I don't think this phase is necessary. Can you check removing it?

Reply:
@drupol : I removed the unpackPhase and then `idb_companion` was not found in the nix-shell.
I think unpacking is essential.